python/lsst/ip/diffim/imageMapReduce.py

```diff
@@ -230,14 +231,15 @@ def run(self, mapperResults, exposure, **kwargs):
         newMI = newExp.getMaskedImage()
 
         reduceOp = self.config.reduceOperation
+        weights = None
```

Comment:
Need to set weights to None so that it is kept in function scope. Moved the setting to `None` to inside the first `if` clause -- when it won't be needed.